### PR TITLE
init code

### PR DESCRIPTION
**Pull request by thesamirkhoury**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "github-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
```
Comment (amirg76) [inline package.json:11]: its nedd to be "types"

**Review by amirg76 — COMMENTED** [review]
see the comment

File: package.json (hunk 1 above)
Comment (amirg76) [inline package.json:11]: check types

**Review by amirg76 — CHANGES_REQUESTED** [review]
check comment

File: package.json (hunk 1 above)
Comment (thesamirkhoury) [reply package.json:11]: fixed

**Review by amirg76 — APPROVED** [review]

**Pull request merged**